pyang/error.py

```diff
@@ -105,6 +105,9 @@ def __init__(self, msg="", exit_code=1):
     'UNEXPECTED_KEYWORD_USES':
       (1,
        'unexpected keyword "%s" under "%s", defined at %s'),
+    'UNEXPECTED_KEYWORD_AUGMENT':
+      (1,
+    'unexpected keyword "%s" augmented under "%s"'),
```

Comment:
Please keep the same indentation.

Reply:
@fredgan,

Thank you for review comment.

Do you mean UNEXPECTED_KEYWORD_AUGMENT defined as the following:

```
    'UNEXPECTED_KEYWORD_AUGMENT':
      (1,
       'unexpected keyword "%s" under "%s", defined at %s'),
```

Thanks

Jie

Reply:
yes

Reply:
Got it. I will update it by your comments.